Kitodo-API/src/main/java/org/kitodo/api/dataformat/PhysicalDivision.java

```diff
@@ -53,6 +53,8 @@ public class PhysicalDivision extends Division<PhysicalDivision> {
      */
     private Map<MediaVariant, URI> mediaFiles = new HashMap<>();
 
+    private MediaPartialView mediaPartialView;
```

Comment:
Basically it is wrong that the `PhysicalDivision` owns the `MediaPartialView`. **Or rather,** if we change the object model in this way, we should remove the view from the reference from the LogicalDivision altogether, and **only** store it here. I understand what you did and we shouldn't change it now. It would be another major change that should be done separately. But, we have to clean that up again.

Reply:
@matthias-ronge I have changed the implementation and removed the inheritance of MediaPartial class from the View.

Reply:
> we should remove the view from the reference from the LogicalDivision altogether

This is something I thought up in the past, but it doesn’t work with the implementation. Please create an issue for this task and assign it to me. I see myself responsible to clean this up.

Reply:
I will create a ticket for you as soon as the changes are in the master. Promise!